src/main/java/Duke.java

```diff
@@ -1,10 +1,67 @@
+package src.main.java;
+import java.util.Scanner;
+
 public class Duke {
+
+    public static final int MAXIMUM_TASK = 100;
+
     public static void main(String[] args) {
+        greetUser();
+        String userInput = "";
+        Task[] userInputList = new Task[MAXIMUM_TASK];
+        do {
+            userInput = scanInput();
+            switch(userInput) {
+                case "list" :
+                    listTasks(userInputList);
+                    break;
+                case "bye" :
+                    System.out.println("Bye. Hope to see you again soon!");
+                    break;
+                default:
+                    if (userInput.contains("done")) {
+                        markedAsDone(userInput, userInputList);
+                    } else {
+                        addTask(userInput, userInputList);
+                    }
+                    break;
+            }
+        } while (!userInput.equalsIgnoreCase("bye"));
+    }
+
+    private static void greetUser() {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        System.out.println("What can I do for you?");
+    }
+
+    private static void addTask(String userInput, Task[] userInputList) {
+        System.out.println("added: " + userInput);
```

Comment:
Maybe it's best to make the letter A in this message uppercase, so it matches the casing of all the other System.out.println messages in the program.